pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -93,96 +88,260 @@ func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 	inspectCmd.Flags().StringVarP(&opts.Hostname, "hostname", "", "", "Configure host to use")
-	cmdutil.StringEnumFlag(inspectCmd, &opts.DigestAlgorithm, "digest-alg", "d", "sha256", []string{"sha256", "sha512"}, "The algorithm used to compute a digest of the artifact")
 	cmdutil.AddFormatFlags(inspectCmd, &opts.exporter)
 
 	return inspectCmd
 }
 
-func runInspect(opts *Options) error {
-	artifact, err := artifact.NewDigestedArtifact(opts.OCIClient, opts.ArtifactPath, opts.DigestAlgorithm)
-	if err != nil {
-		return fmt.Errorf("failed to digest artifact: %s", err)
-	}
+type BundleInspectResult struct {
+	InspectedBundles []BundleInspection `json:"inspectedBundles"`
+}
+
+type BundleInspection struct {
+	Authentic              bool                  `json:"authentic"`
+	Certificate            CertificateInspection `json:"certificate"`
+	TransparencyLogEntries []TlogEntryInspection `json:"transparencyLogEntries"`
+	SignedTimestamps       []time.Time           `json:"signedTimestamps"`
+	Statement              *in_toto.Statement    `json:"statement"`
+}
+
+type CertificateInspection struct {
+	certificate.Summary
+	NotBefore time.Time `json:"notBefore"`
+	NotAfter  time.Time `json:"notAfter"`
+}
 
-	opts.Logger.Printf("Verifying attestations for the artifact found at %s\n\n", artifact.URL)
+type TlogEntryInspection struct {
+	IntegratedTime time.Time
+	LogID          string
+}
 
+func runInspect(opts *Options) error {
+	// 1. Load the bundles
+	// 2. Parse & check the "authenticity" of the bundle
+	// 3. Summarize the certs
 	attestations, err := verification.GetLocalAttestations(opts.BundlePath)
 	if err != nil {
-		return fmt.Errorf("failed to read attestations for subject: %s", artifact.DigestWithAlg())
+		return fmt.Errorf("failed to read attestations")
 	}
 
-	policy, err := buildPolicy(*artifact)
-	if err != nil {
-		return fmt.Errorf("failed to build policy: %v", err)
-	}
+	inspectedBundles := []BundleInspection{}
+	unsafeSigstorePolicy := verify.NewPolicy(verify.WithoutArtifactUnsafe(), verify.WithoutIdentitiesUnsafe())
 
-	results, err := opts.SigstoreVerifier.Verify(attestations, policy)
-	if err != nil {
-		return fmt.Errorf("at least one attestation failed to verify against Sigstore: %v", err)
-	}
+	for _, a := range attestations {
+		inspectedBundle := BundleInspection{}
 
-	opts.Logger.VerbosePrint(opts.Logger.ColorScheme.Green(
-		"Successfully verified all attestations against Sigstore!\n\n",
-	))
+		// we ditch the verificationResult to avoid even implying that it is "verified"
+		// you can't meaningfully "verify" a bundle with such an Unsafe policy!
+		_, err := opts.SigstoreVerifier.Verify([]*api.Attestation{a}, unsafeSigstorePolicy)
 
-	// If the user provides the --format=json flag, print the results in JSON format
-	if opts.exporter != nil {
-		details, err := getAttestationDetails(opts.Tenant, results)
+		// TODO: if the err is present, we keep on going because we want to be able to
```

Comment:
**Is this something to address in this PR, remove, or follow up later?**

If later follow up, just wanting to make sure there is an issue for this.

Reply:
this is more food for thought for future iterations than a strict todo & i've adjusted the comment accordingly.